docs/design/StratisSoftwareDesign.lyx

```diff
@@ -7210,12 +7173,26 @@ Encrypted device creation
 
 \begin_layout Standard
 Stratis does not wipe the device prior to initializing it with a LUKS2 header.
- Due to how data is encrypted with LUKS2, if a user is concerned with an
- attacker being able to determine exactly how much of the data on the device
- is encrypted, the device should be wiped with random data prior to providing
- it to a Stratis pool.
+ This means that an attacker can potentially deduce the following from examing
+ a disk that has been initialized using Stratis without any prior preparation
+ outside of Stratis:
```

Comment:
You can leave out "outside of Stratis", I think that's clear from "prior preparation".